src/Umbraco.Web.UI/Umbraco/config/lang/en_us.xml

```diff
@@ -1363,6 +1363,17 @@ To manage your website, simply open the Umbraco backoffice and start adding cont
     <key alias="pickedTrashedItem">You have picked a media item currently deleted or in the recycle bin</key>
     <key alias="pickedTrashedItems">You have picked media items currently deleted or in the recycle bin</key>
     <key alias="trashed">Trashed</key>
+    <key alias="openMedia">Open media</key>

```

Comment:
I think we should consider making this more clear. Maybe something like "Open in media section".

Reply:
Wording is changed to:
    <key alias="openMedia">Open in Media Library</key>
    <key alias="changeMedia">Change Media Item</key>